src/libpsl-native/src/setdate.h

```diff
@@ -17,6 +17,6 @@ typedef struct setDateInfo
     int32_t DST;
 } SetDateInfo;
 
-int32_t SetDate(const SetDateInfo &info);
+int32_t SetDate(const SetDateInfo info);
 
```

Comment:
Code size can have a bigger impact then optimizing stuff like this, and in this case, 8 pushes are more code than 1, so it's sort of a win/win either way.

But I don't understand why you'd have further marshalling changes - the reference was marshaled as a pointer before, so did you undo that?